### PR TITLE
Update opengapps.xml

### PR DESCRIPTION
**Pull request by darran-kelinske-fivestars**
Update references to opengapps

### CODE REVIEW

File: opengapps.xml [hunk 1]
```diff
@@ -1,13 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <manifest>
   <remote name="opengapps" fetch="https://github.com/opengapps/" />
```
Comment (ayufan) [inline opengapps.xml:3]: Why we do not change this?

**Review by ayufan — CHANGES_REQUESTED** [review]

File: opengapps.xml (hunk 1 above)
Comment (darran-kelinske-fivestars) [reply opengapps.xml:3]: some of the repos are still hosted on github

Comment (ayufan) [reply opengapps.xml:3]: Hmm, OK :)

Then we should make `opengapps-gitlab`, as `gitlab` is just too generic.

Comment (darran-kelinske-fivestars) [reply opengapps.xml:3]: see here: https://github.com/opengapps/aosp_build

**Review by ayufan — CHANGES_REQUESTED** [review]

File: opengapps.xml (hunk 1 above)
Comment (darran-kelinske-fivestars) [reply opengapps.xml:3]: done

**Review by ayufan — APPROVED** [review]

**Comment by darran-kelinske-fivestars** [comment]
Is there a build box that will build with the updates?

**Pull request merged**